### PR TITLE
Fix NullPointerException

### PR DESCRIPTION
**Pull request by bparrishMines**
Fixes https://github.com/flutter/flutter/issues/78405

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [relevant style guides] and ran [the auto-formatter]. (Note that unlike the flutter/flutter repo, the flutter/plugins repo does use `dart format`.)
- [x] I signed the [CLA].
- [x] The title of the PR starts with the name of the plugin surrounded by square brackets, e.g. `[shared_preferences]`
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I [updated pubspec.yaml](https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#version-and-changelog-updates) with an appropriate new version according to the [pub versioning philosophy].
- [x] I updated CHANGELOG.md to add a description of the change.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [ ] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test exempt.
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/plugins/blob/master/CONTRIBUTING.md
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[relevant style guides]: https://github.com/flutter/plugins/blob/master/CONTRIBUTING.md#style
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[the auto-formatter]: https://github.com/flutter/plugins/blob/master/script/tool/README.md#format-code

### CODE REVIEW

**Comment by bparrishMines** [comment]
The PR should ideally fix the crash, but I'm not sure how to test it. I was unable to reproduce the bug. The closes I was able to get was:

```java
  @Test
  public void runnableChecksContainerViewIsNull() {
    final ActivityScenario<WebViewTestActivity> scenario =
        ActivityScenario.launch(WebViewTestActivity.class);
    scenario.onActivity(activity -> {
      final TestView testContainerView = new TestView(activity);

      final InputAwareWebView webView = new InputAwareWebView(activity, testContainerView);
      activity.setContentView(webView);

      final TestView testProxyAdapterView = new TestView(activity);
      webView.checkInputConnectionProxy(testProxyAdapterView);
      webView.setContainerView(null);

      assertNotNull(testContainerView.postAction);

      InputMethodManager imm =
          (InputMethodManager) activity.getSystemService(INPUT_METHOD_SERVICE);
      imm.dispatchKeyEventFromInputMethod(webView, new KeyEvent(KeyEvent.ACTION_DOWN, KeyEvent.KEYCODE_0) );

      testContainerView.postAction.run();
    });
  }
```

But this still doesn't reproduce it :\

File: packages/webview_flutter/webview_flutter/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
-## NEXT
+## 2.0.12
 
 * Improved the documentation on using the different Android Platform View modes.
+* Fix `NullPointerException` on Android from a race condition when changing focus. This only effects
```
Comment (stuartmorgan) [inline packages/webview_flutter/webview_flutter/CHANGELOG.md:4]: Nit: affects

**Review by stuartmorgan — COMMENTED** [review]
Presumably the test doesn't work because there's a race between the Runnable and the test code? If that's it, could you introduce a test-only delay of processing that Runnable to ensure that `setContainerView(null)` has time to run?

**Review by blasten — APPROVED** [review]
LGTM %  nit above

**Comment by stuartmorgan** [comment]
@bparrishMines Ping? What's the status on this crash fix?

**Pull request merged**